### ISSUE TITLE
Education - Where did ye go!?

### ISSUE THREAD

**Issue by thisisjoewells** [TheDetroitZoo/dzs-import#5]
As discussed on Slack...

The education menu has disappeared mysteriously from the top-nav, and footer.  

What gives?  No one knows yet.  I wanted to officially log the issue, since I don't think anyone has....

This of course needs to be put back by 9/1, for among other reasons, so we can finish fixing the styling/media-query issues with the top-nav.

I will go ahead and put int just a top-level 'Education' button so we can visualize things in the meantime.

**Comment by thisisjoewells** [comment]
A hypothesis!

When Kyle and I imported the Education pages over from Joomla some pages failed to overwrite the placeholders I had made.  

I had made some empty placeholder pages, at CJ's request so he could visualize the menu structure.

I think I deleted the Education Placeholder page.  

That would make the menu item break or disappear, could it have removed everything underneath it too?  That might be what happened!
